huaweicloud/services/ccm/resource_huaweicloud_ccm_private_certificate.go

```diff
@@ -0,0 +1,443 @@
+package ccm
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceCcmPrivateCertificate() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCcmPrivateCertificateCreate,
+		ReadContext:   resourceCcmPrivateCertificateRead,
+		DeleteContext: resourceCcmPrivateCertificateDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"issuer_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"key_algorithm": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"signature_algorithm": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"distinguished_name": {
```

Comment:
can we get the distinguished_name is not specified? if yes, please add **Computed** attr.